packages/sdk/src/endpoints/AttributesEndpoint.ts

```diff
@@ -2,19 +2,39 @@ import {
     ConnectorAttribute,
     ConnectorAttributes,
     ConnectorResponse,
-    CreateAttributeRequest,
+    CreateIdentityAttributeRequest,
+    CreateRelationshipAttributeRequest,
     ExecuteIQLQueryRequest,
     ExecuteIdentityAttributeQueryRequest,
     ExecuteRelationshipAttributeQueryRequest,
     ExecuteThirdPartyRelationshipAttributeQueryRequest,
     GetAttributesRequest,
-    GetValidAttributesRequest
+    GetValidAttributesRequest,
+    NotifyPeerAboutIdentityAttributeSuccessionRequest,
+    SucceedIdentityAttributeRequest,
+    SucceedRelationshipAttributeRequest
 } from "../types";
 import { Endpoint } from "./Endpoint";
 
 export class AttributesEndpoint extends Endpoint {
-    public async createAttribute(request: CreateAttributeRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
-        return await this.post("/api/v2/Attributes", request);
+    public async createIdentityAttribute(request: CreateIdentityAttributeRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
+        return await this.post("/api/v2/Attributes/IdentityAttribute", request);
+    }
+
+    public async createRelationshipAttribute(request: CreateRelationshipAttributeRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
```

Comment:
Is there a reason this isnt' called `createAndShareRelationshipAttribute`. All of the new connector routes are named according to their use cases, except for this one. Obviously, this isn't a hard requirement, I'm just curious.